### DISCUSSION TITLE
ssh-know-host-cm configmap is overwritten

### DISCUSSION THREAD

**Discussion by sunmd01** [argoproj/argo-cd#10035] [Q&A] [answered]
When I use this command to upgrade ArgoCD kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/<version>/manifests/install.yaml , ssh-know-host-cm ConfigMap is overwritten.

After I upgraded, I had to update ssh-know-host again.

Is there a better way for me to avoid performing the operation?

**Comment by crenshaw-dev** [comment, marked as answer]
I'd recommend creating your own git repo to hold your Argo CD manifests. You can reference the external (https) install manifests and then apply a Kustomize overlay to update your ssh-known-host-cm.
